vendor/bat-native-ads/src/bat/ads/internal/ml/data/vector_data_unittest.cc

```diff
@@ -31,7 +31,7 @@ TEST_F(BatAdsVectorDataTest, DenseVectorDataInitialization) {
 
```

Comment:
We should probably add some tests for the new sum, divide and non zero elements functionality

Reply:
We should add tests to cover all known cases

Reply:
Added unit tests in vector_data_unittest.cc